conseil-api/src/main/scala/tech/cryptonomic/conseil/api/Conseil.scala

```diff
@@ -16,11 +16,14 @@ import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor}
 import scala.language.postfixOps
 import scala.util.Failure
 
-object Conseil extends App with LazyLogging with ConseilAppConfig with FailFastCirceSupport with ConseilMainOutput {
+object Conseil extends App with ConseilAppConfig with FailFastCirceSupport with ConseilMainOutput {
+
+  Logging.init()
 
   loadApplicationConfiguration(args) match {
     case Left(errors) =>
-    //nothing to do
+      //nothing to do
+      logger.error("There is an error in the provided configuration")
```

Comment:
can we get errors from `Left` logged here?

Reply:
IIRC, no need to do it here because they've been already logged by the `loadApplicationConfiguration` call...